ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/Wei.java

```diff
@@ -18,7 +18,7 @@
 
 import java.math.BigInteger;
 
-import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.bytes.Bytes;
```

Comment:
Why this change is necessary ?

Reply:
In `TransactionRLPDecoder`, we previously had `Wei.of` of a `BigInteger`, sometimes that `BigInteger` was negative so the `Wei` was negative. When I changed it to `Wei.wrap` I needed to allow `Bytes` instead of just `Bytes32` since it could be shorter.

Reply:
That makes sense